### DISCUSSION TITLE
Add ability to remove organisation authorisation from app.

### DISCUSSION THREAD

**Discussion by dbut2** [community/community#59310] [Mobile] [answered]
<details>
<summary>
Organisation adding an IP allow-list while user is authorised in app, causes an error to show up every time the app is opened, user doesn't currently have a way to remove this authorisation from the app.
</summary>
GitHub for iOS, Version 1.118.1 (103027020)
Device: iPhone15,3 (iOS 16.5.1)
</details>

**Comment by stevepopovich** [comment]
Hi @dbut2! I would love some clarity here. Are you enterprise users? Do you have a screenshot of the behavior? Thanks for writing in!

**Reply by dbut2** [reply]
Hi @stevepopovich, thanks for the reply. I am an enterprise user, and here is screenshot of error that shows (org name and ip redacted).

![IMG_6923](https://github.com/community/community/assets/61171623/ee6a6b70-eda1-4589-a0ba-5736bf8991f4)

Just to clarify, I think this is expected behaviour as I'm not using a now allow-listed IP address, but would rather the ability to just remove the organisation auth :)

**Reply by dbut2** [reply]
Also, unclear if related but I am unable to use "Issues" and "Pull Request" pages

![IMG_6924](https://github.com/community/community/assets/61171623/d828fed7-fa0b-435a-b607-37c476a3dc9e)

**Reply by stevepopovich** [reply]
Yeah, I would say those things are related. If the admin of the enterprise has a whitelist enabled, you would need to be on that whitelist. I don't think there is a way for you personally to circumvent that org auth and I would expect nothing to work in the app while you are logged in with that enterprise account and not whitelisted. 

I def need to do some more digging here, but those are my hunches. I will double check this isn't a bug on our end 👍🏼

**Reply by dbut2** [reply]
I think I may have misunderstood enterprise before, apologies. This is a github.com account that is authorised to an organisation under what is listed as an enterprise, but not on an enterprise server. I do also have personal work on this account not related to the org.

**Reply by stevepopovich** [reply]
Oh, so you are logged in as a normal personal account. Gotcha. So being apart of this org is blocking you from using the rest of the app?

**Reply by dbut2** [reply]
Just Issues and Pull Requests, the rest of the app is fine from what I can tell aside from the warning that shows at the bottom

**Reply by stevepopovich** [reply]
Oh, very interesting. This is sounding more like a bug on our end, and less of a setting that needs fixed. Ok, let me look into this.

**Reply by stevepopovich** [reply, marked as answer]
Ok, found out this is a known issue that hasn't been fixed. We will look into prioritizing this. Thanks!

**Reply by Mikegoossens1980** [reply]
> Hi @dbut2! I would love some clarity here. Are you enterprise users? Do you have a screenshot of the behavior? Thanks for writing in!

- - [x] - @
